.github/workflows/ci.yml

```diff
@@ -0,0 +1,76 @@
+name: CI
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  check_clippy:
+    name: Check clippy for Rust demo apps
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        rust: [stable]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install ${{ matrix.rust }}
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.rust }}
+          override: true
+      - name: Check clippy
+        run: cargo +${{ matrix.rust }} clippy --all-targets --all-features -- -D warnings
+
+  check_fmt:
+    name: Check formatting for Rust demo apps
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        rust: [stable]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install ${{ matrix.rust }}
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.rust }}
+          override: true
+      - name: Check formatting
+        run: cargo +${{ matrix.rust }} fmt -- --check
+
+  test_mysql_cdc:
+    name: Run tests for MySQL CDC app
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        rust: [stable]
+    steps:
+      - name: Install Fluvio Local Cluster
+        uses: infinyon/fluvio@master
+        with:
+          cluster-type: local
+      - name: Install MySQL containers
+        run: |
+          pwd
+          pushd cdc-mysql/docker
+          ./install.sh -n mysql-producer -d ~/mysql-cdc/mysql-producer -p 3080
+          ./install.sh -n mysql-consumer -d ~/mysql-cdc/mysql-consumer -p 3090
+          popd
+      - name: Run CDC Producer
+        run: cargo run --bin cdc-producer -- mysql-cdc/producer_profile.toml &
```

Comment:
move the testing in a separate script or Makefile.  this allow other to run in their machine

Reply:
better yet, can you put as Rust integration test